src/frontend/src/env/networks.icrc.env.ts

```diff
@@ -464,9 +460,17 @@ export const ICRC_TOKENS: IcInterface[] = [
 	...(nonNullish(CKXAUT_IC_DATA) ? [CKXAUT_IC_DATA] : [])
 ];
 
-export const ICRC_TOKENS_LEDGER_CANISTER_IDS: LedgerCanisterIdText[] = ICRC_TOKENS.map(
-	({ ledgerCanisterId }) => ledgerCanisterId
-);
+const ADDITIONAL_ICRC_TOKENS: IcInterface[] = [...(nonNullish(BURN_IC_DATA) ? [BURN_IC_DATA] : [])];
+
+export const ICRC_TOKENS: IcInterface[] = [
+	...PUBLIC_ICRC_TOKENS,
```

Comment:
Should we also rename this to PUBLIC_CK_ICRC_TOKENS as it only holds CKBTC, CKETH and CKUSDC at the moment?